packages/xen-api/src/_Ref.js

```diff
@@ -5,6 +5,11 @@ export default {
   // Reference to use to indicate it's not pointing to an object
   EMPTY,
 
+  // Whether this value is a null XAPI ref
+  isEmpty(val) {
+    return val === EMPTY
+  },
```

Comment:
I'm not found of `isEmpty` not strictly the negation of `isNotEmpty` :grimacing: 

Maybe you should simply test `=== Ref.EMPTY` in your code.

Reply:
It actually makes sense, though. Because it implicitly means "is an empty ref" and "is not an empty ref". So both can be false. And that's actually why I had a bug in my code: I wrongly assumed that `!Ref.isNotEmpty(...)` would tell me if I had an empty opaque ref.

I'm ok with `=== Ref.EMPTY`, though.

Reply:
I see your point.

I don't think we need a dedicated method for such a trivial test though, let's go with explicit equality testing :slightly_smiling_face: